src/sbt-test/settings/lazy-protoc-resolution/build.sbt

```diff
@@ -1 +1,20 @@
-PB.protocDependency := "non" % "existing" % "artifact"
+// PB.targets, no proto
+lazy val a = project
+  .settings(
+    PB.protocVersion := "1.2.3+non-existing-a",
+    Compile / PB.targets := Seq(PB.gens.java -> (Compile / sourceManaged).value)
+  )
+
+// no PB.targets, one proto
+lazy val b = project
+  .settings(
+    PB.protocVersion := "1.2.3+non-existing-b",
+  )
+
+// PB.targets, one ignored proto
```

Comment:
new speculative scenario, making sure conditions for fetching protoc and running it are aligned